utopian-io/v2.utopian.io
# Override the token received from GitHub when a user logs in

### Verify that

### My PR doesn't concern a Jira task
We need to udate the access token when a user logs in

### **Test plan**

_Update the access token_

* Check the github token in the database
* Log out and then log in
* Go to the create project page and add one of your repo

**Expected result:** The access token has been updated. You can add your repo in the create project page.
